configure.ac

```diff
@@ -55,24 +55,9 @@ GETTEXT_PACKAGE=intel_lpmd
 AC_SUBST([GETTEXT_PACKAGE])
 AC_DEFINE_UNQUOTED([GETTEXT_PACKAGE], ["$GETTEXT_PACKAGE"], [Gettext package])
 
-dnl
-dnl Checks for new dbus-glib property access function
-dnl
-PKG_CHECK_MODULES([DBUS_GLIB], [dbus-glib-1 >= 0.94],
-	[dbus_glib_global_set_disable_legacy_property_access],
-	[ac_have_dg_prop="1"], [ac_have_dg_prop="0"]
-)
-AC_DEFINE_UNQUOTED([HAVE_DBUS_GLIB_DISABLE_LEGACY_PROP_ACCESS],
-	[$ac_have_dg_prop],
-	[Define if you have a dbus-glib with dbus_glib_global_set_disable_legacy_property_access()]
-)
-
-PKG_CHECK_MODULES([DBUS], [dbus-1 >= 1.1 dbus-glib-1 >= 0.94])
-
 GLIB_VERSION_DEFINES="-DGLIB_VERSION_MIN_REQUIRED=GLIB_VERSION_2_26"
-DBUS_CFLAGS="$DBUS_CFLAGS $GLIB_VERSION_DEFINES"
 
-PKG_CHECK_MODULES([GLIB], [gio-unix-2.0 >= 2.22 gmodule-2.0])
+PKG_CHECK_MODULES([GLIB], [gio-unix-2.0 >= 2.22 gmodule-2.0 glib-2.0 >= 2.80 gobject-2.0])
```

Comment:
Why is glib 2.80 required?

Reply:
Thank you for reviewing it. 

I think glib-2.0 is enough. I'll push a new version. :)

Reply:
Technically glib and gobject don't need to be explicit, as gio and gmodule automatically include those by definition.  But ultimately that is a style question that will be up to the maintainers.

Reply:
"glib-2.0  gobject-2.0" are for compiling tools/intel_lpmd_control.c.